dask/dataframe/io/parquet.py

```diff
@@ -121,7 +117,8 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                         columns=[c for c in pf.columns if c in dtypes])
 
     for cat in categories:
```

Comment:
Right, got you. They should not appear, but I'm not convinced it should be an exception.